sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **Breaking changes**
 - Removed property `length` from `CategorizedEntity`, `SentenceSentiment`, `LinkedEntityMatch`, `AspectSentiment`, `OpinionSentiment`, and `PiiEntity`.
 To get the length of the text in these models, just call `len()` on the `text` property.
+- When a parameter or endpoint is not compatible with the API version you specify, we will now return a `ValueError` instead of a `NotImplementedError`.
```

Comment:
Do we want to mention the change in client side validation?

Reply:
Oh, that's a great point, going to add that to the changelog